_posts/patterns/2015-12-11-tables.md

```diff
@@ -0,0 +1,257 @@
+---
+layout:         pattern
+title:          Tables
+date:           2015-12-11 00:00:00
+
+categories:     patterns
+tags:
+- atomic
+- pattern
+- buttons
+
+slug:           tables
+
+url_styles:     "components/_tables.scss"
+
+description:    A collection of buttons available for using within the edX platform.
```

Comment:
NIT: Mind updating the description? You can generally take the ``info`` field and simplify it. I don't think we're using it anywhere (and I should remove it from all patterns), so you could alternatively just remove it from this pattern's metadata.